contrib/clojure-package/examples/infer/objectdetector/test/infer/objectdetector_example_test.clj

```diff
@@ -0,0 +1,66 @@
+;; Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
This is good too. I would keep all these as integration tests as they make sure the examples will run ok.
It would be nice to merge with master and see if the integration test script runs ok with these. We may need to tweak and put some script downloading stuff in.